src/api/config.i

```diff
@@ -0,0 +1,34 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
this file is generated by cmake?
if yes, it should not be put into git repo.

Reply:
Yes, this is generated directly from "config.i.in" by cmake.
I have deleted this file "config.i"